ios/Classes/PdftronFlutterPlugin.m

```diff
@@ -696,6 +752,8 @@ - (void)handleOpenDocumentMethod:(NSDictionary<NSString *, id> *)arguments resul
 
     [PTOverrides overrideClass:[PTDocumentViewController class] withClass:[PTFlutterViewController class]];
     
+    [PTOverrides overrideClass:[PTThumbnailsViewController class] withClass:[FLThumbnailsViewController class]];
```

Comment:
These only need to be called once, not every time a tab is created. And this looks to be too late in the lifecycle to make the call to override the `PTDocumentViewController`. Have you tested this?

Reply:
I have tested and it works, since the controllers are allocated and initiated after the override call. However yes, these need to be called only once, so I have moved them in a new commit.